pkg/util/ssh.go

```diff
@@ -142,21 +142,42 @@ func (s *SSHTunnel) Close() error {
 	return nil
 }
 
+// Interface to allow mocking of ssh.Dial, for testing SSH
+type sshDialer interface {
+	Dial(network, addr string, config *ssh.ClientConfig) (*ssh.Client, error)
+}
+
+// Real implementation of sshDialer
+type realSSHDialer struct{}
```

Comment:
nit, it's traditional to add a 

```
var _ sshDialer = realSSHDialer
```

to ensure the contract stays implemented :)